admin_guide/out_of_resource_handling.adoc

```diff
@@ -64,8 +68,7 @@ Each threshold is of the following form:
 <eviction_signal><operator><quantity>
 ----
 
-* Valid `eviction-signal` tokens as defined in
-xref:out-of-resource-eviction-signals[Eviction Signals].
+* Valid `eviction-signal` tokens as defined by eviction signals.
```

Comment:
its meant to mean the signals in table 1 (which now lists one), but in 3.4 needs to list 4 more (all disk related)